### ISSUE TITLE
Add User-Configurable Jump-Back When Changing Playback Devices or Resuming After Long Pause

### ISSUE THREAD

**Issue by eroc1990** [podverse/podverse-rn#1217]
**Is your feature request related to a problem? Please describe.**
Currently, when changing playback devices (switching from earbuds to car audio, or vice versa), I need to manually jump back 10-15 seconds, since my earbuds don't pause when removing them from my ears. Alternately, it's easy to forget what I already listened to after having paused the podcast for an extended period of time.

**Describe the solution you'd like**
Set an automatic skip back (user configurable, and toggleable at an app level if the feature is not wanted by the end user) of a user-defined time when resuming playback after switching playback devices or after resuming the podcast after an extended pause.

**Describe alternatives you've considered**
Manually jumping back obviously works, but other popular podcast players feature similar functionality.

**Comment by mitchdowney** [comment]
@eroc1990 interesting feature request. Seems like it might be tricky for us to make sure it only rewinds on the proper events...but should be doable. I'm going to mark this as blocked for now though, until we update the audio player library we're using. Anything related to audio events and time jumps I think we should wait until after the update #1179

**Comment by eroc1990** [comment]
Makes sense. Thanks for considering implementing this. Would save me (and I'm sure others) would appreciate not having to seek back as often.

**Comment by mitchdowney** [comment]
We are archiving all Github issues and PRs that pertain to the previous generation Podverse. The Podverse dev team is currently 100% focused on the next generation app, and will be only working on next gen Github issues going forward.

If you would like to see a work-in-progress demo of the upcoming Podverse website, please visit https://alpha.podverse.fm. Our tentative release date is Spring 2026.

If you're a developer who may be interested in contributing to Podverse Next-Gen, please see our [Local Dev Setup documentation](https://github.com/podverse/podverse-ops/blob/v5-develop/dev/local-dev-setup.md). We can also be reached through [Matrix or Discord](https://podverse.fm/contact).

I'm sorry for the abrupt announcement, but thank you for your bug reports and feature requests.
